### PR TITLE
Move ntp tooltip closer to cell

### PR DESCRIPTION
**Pull request by ecotopian**
https://ecosia.atlassian.net/browse/MOB-1499

### CODE REVIEW

File: Client/Ecosia/UI/NTP/NTPLayout.swift [hunk 1]
```diff
@@ -36,7 +36,7 @@ class NTPLayout: UICollectionViewFlowLayout {
                     let font = UIFont.preferredFont(forTextStyle: .callout)
                     let height = text.heightWithConstrainedWidth(width: impact.bounds.width - 16 - 48, font: font) + 24 + 16
                     tooltip.frame.size.height = height
-                    tooltip.frame.origin.y -= (height - 32)
+                    tooltip.frame.origin.y -= (height - 40)
```
Comment (jman82) [inline Client/Ecosia/UI/NTP/NTPLayout.swift:39]: I wonder if we would benefit from static constant declaration for these values (40). Same for line 37, for example, where there are operations like .width -16 -48, maybe having them as constants would increase readability?

**Review by vauxhall — APPROVED** [review]
Looking great ✨

File: Client/Ecosia/UI/NTP/NTPLayout.swift [hunk 2]
```diff
@@ -34,9 +34,9 @@ class NTPLayout: UICollectionViewFlowLayout {
 
                 if let text = highlightDataSource?.ntpLayoutHighlightText() {
                     let font = UIFont.preferredFont(forTextStyle: .callout)
-                    let height = text.heightWithConstrainedWidth(width: impact.bounds.width - 16 - 48, font: font) + 24 + 16
+                    let height = text.heightWithConstrainedWidth(width: impact.bounds.width - 4 * NTPTooltip.margin, font: font) + 2 * NTPTooltip.containerMargin + NTPTooltip.margin
```
Comment (jman82) [inline Client/Ecosia/UI/NTP/NTPLayout.swift:37]: :heart:

**Review by jman82 — APPROVED** [review]
:pray: :tropical_drink:

**Pull request merged**